team_b/yappy/lib/industry_menu.dart

```diff
@@ -12,20 +12,19 @@ import 'services/database_helper.dart';
 import 'services/file_handler.dart';
 import 'services/model_manager.dart';
 import 'services/speech_state.dart';
+import 'package:flutter/services.dart';
+import 'package:file_picker/file_picker.dart';
 
 class IndustryMenu extends StatefulWidget {
   final String title;
   final IconData icon;
   final SpeechState speechState;
   final ModelManager modelManager;
 
-  const IndustryMenu({
-    required this.title, 
-    required this.icon,
-    required this.speechState,
-    required this.modelManager,
-    super.key
-  }); 
+  IndustryMenu({required this.title, required this.icon, super.key});
+
```

Comment:
Removed speechState and modelManager, those are still needed.

Also, FileName and FileContent seem like they're in the wrong scope? Lower down, when you use them it's saying the variables don't exist.